debian/control

```diff
@@ -4,7 +4,8 @@ Priority: optional
 Maintainer: EndlessM Maintainers <maintainers@endlessm.com>
 Uploaders: Sjoerd Simons <sjoerd.simons@collabora.co.uk>, Hector Oron <hector.oron@collabora.co.uk>, Srdjan Grubor <sgnn7@sgnn7.org>
 Standards-Version: 3.9.3
-Build-Depends: debhelper (>= 9)
+Build-Depends: debhelper (>= 9),
+               meson,
```

Comment:
I think you should include `libdpkg-perl` here even though it's an implicit dependency of `debhelper`.